app/__mocks__/recoil.ts

```diff
@@ -15,7 +15,7 @@ export function setMockAtoms(newMockValues: { [key: string]: any }) {
 export const getValue = (atom) => {
   if (mockValuesStore[atom.key]) {
     const str = JSON.stringify(atom.params);
-    if (mockValuesStore[atom.key].hasOwnProperty(str)) {
+    if (mockValuesStore[atom.key].hasOwn(str)) {
```

Comment:
Is `hasOwn` a thing? I could be wrong but I remember something like `Object.hasOwn(some_object,'some_key')`

Reply:
Fixed! Thank you 🙏